src/construction.cpp

```diff
@@ -1500,3 +1500,24 @@ std::vector<std::string> construction::get_folded_time_string( int width ) const
     std::vector<std::string> folded_time = foldstring( time_text, width );
     return folded_time;
 }
+
+void finalize_constructions()
+{
+    for( construction &con : constructions ) {
+        if( con.post_special == &construct::done_vehicle ) {
+            std::set<itype_id> frames;
+            for( auto &vp : vpart_info::get_all() ) {
```

Comment:
This loop gets executed and `frames` gets constructed every time `done_vehicle` is found, but neither of them depend on `con` at all. We _know_ that it happens only once (for now?), but why is this inside anyways?